tests/acceptance/features/apiGraph/createUser.feature

```diff
@@ -0,0 +1,53 @@
+@api @skipOnOcV10
+Feature: create user
+  Only user with admin permissions can create new user
+
+  Note - this feature is run in CI with ACCOUNTS_HASH_DIFFICULTY set to the default for production
+  See https://github.com/owncloud/ocis/issues/1542 and https://github.com/owncloud/ocis/pull/839
+
+  Background:
+    Given user "Alice" has been created with default attributes and without skeleton files
+
```

Comment:
Need one more line space, referring to documentation https://github.com/owncloud/docs-server/pull/697/files#diff-12a0f404a5bbbda65e50018a487cd725797106d670f4308b921d2081dc4b3516R1129-R1154:~:text=more%20relevant%20discussion.-,%3D%3D%3D%3D%20Inter%2Dscenario%20spacings,%2D%2D%2D%2D,-Commenting%20on%20lines